content/src/gradient.rs

```diff
@@ -104,13 +103,16 @@ impl Gradient {
         let transform = Transform2F::default();
         Gradient {
             geometry: GradientGeometry::Radial { line: line.to_line(), radii, transform },
-            stops: SortedVector::new(),
+            stops: Vec::new(),
         }
     }
 
     #[inline]
     pub fn add(&mut self, stop: ColorStop) {
-        self.stops.push(stop);
+        let index = self.stops.binary_search_by(|other| {
+            other.offset.partial_cmp(&stop.offset).unwrap()
+        }).unwrap_or_else(convert::identity);
+        self.stops.insert(index, stop);
```

Comment:
Wouldn't it be faster to keep the old sorted vector code since `insert` has to be `O(n)` anyway?

Reply:
This is almost the same code `SortedVector` used, just inlined.

I did change `unwrap_or(Less)` to just `unwrap()`, and am sorting only on `offset`, which now that I think about it might not be valid. What's the expected behaviour for gradients with two points at the same offset?

Reply:
Here's what the [spec](https://html.spec.whatwg.org/multipage/canvas.html#interpolation) says:

> If multiple stops are added at the same offset on a gradient, then they must be placed in the order added, with the first one closest to the start of the gradient, and each subsequent one infinitesimally further along towards the end point (in effect causing all but the first and last stop added at each point to be ignored).

Reply:
Ah, then both versions are actually wrong, since `binary_search` can return ‘any one of the matches’ to insert at. I'll add a fix.